### ISSUE TITLE
Change default for tls_method from TLSv1 to TLSv1.2+

### ISSUE THREAD

**Issue by dilyanpalauzov** [kamailio/kamailio#3301]
The default value for `tls_method` is `TLSv1`: https://kamailio.org/docs/modules/devel/modules/tls.html#tls.p.tls_method .  This value is suboptimal as default.

Please change the default value to `TLSv1.2+` and remove `modparam("tls", "tls_method", …)` from misc/examples/outbound/edge_websocket.cfg and misc/examples/webrtc/websocket.cfg .

**Comment by henningw** [comment]
Yes, it make sense to change the default to not allow obselete/deprecate algorithms. Regarding the change in the examples, I think this can stay, it does not hurt and they are used from people also with older versions in the end.
